tests/integration/utils.py

```diff
@@ -90,6 +90,17 @@ def build_image_for_test_case(tmp_path: Path, containerfile: str, test_case: str
         "--network",
         "none",
     ]
+
+    # this should be extended to support more archs when we have the means of testing it in our CI
+    rpm_repos_path = f"{tmp_path}/{test_case}-output/deps/rpm/x86_64/repos.d"
+    if Path(rpm_repos_path).exists():
+        cmd.extend(
+            [
+                "-v",
+                f"{rpm_repos_path}:/etc/yum.repos.d",
```

Comment:
> Do we want to add a :Z here?

Hmm, this runs as part of GH actions on Ubuntu latest right? In which case we don't, because Ubuntu doesn't have SELinux, they use Apprmor.
Since you've mentioned it I think it makes more sense to include `ro` as  mount option in there for added safety.

Reply:
Does it make sense to keep the `:Z` since it could benefit local environments? (I run this on Fedora)

Reply:
Yeah, I was thinking about local environments here

Reply:
As for `ro`, I am wondering if we really need it, as this is supposed to be ephemeral test data.

If we do want to add it, I think it makes sense to add it to the other volume mount as well (cachi2-output).